argoverse/evaluation/detection/utils.py

```diff
@@ -319,27 +321,24 @@ def filter_instances(
     return filtered_instances
 
 
-def rank(dts: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
-    """Get the rankings for the detections, according to detector confidence.
+def rank(label_records: np.ndarray) -> np.ndarray:
+    """Rank the `ObjectLabelRecord` objects in descending order by score.
 
     Args:
-        dts: Detections (N,).
+        label_records: Array of `ObjectLabelRecord` objects. (N,).
```

Comment:
thanks for the updates, @benjaminrwilson. I liked the old name `dts`, more, though. Since it conveys the semantics. `label_records` sounds like GT labels, which they are not